sources/editor/Stride.Core.Assets.Editor/ViewModel/AssetCollectionViewModel.cs

```diff
@@ -153,6 +154,18 @@ public override int GetHashCode()
             }
         }
 
+        [DataContract(nameof(AssetFilterViewModelData))]
+        public sealed class AssetFilterViewModelData
+        {
+            public string DisplayName = "";
+            public string Filter = "";
+            public bool IsActive = false;
+            public FilterCategory category = FilterCategory.AssetName;
+        }
+
+        // Storing out primitive data for filters to save between instances of editor
+        List<AssetFilterViewModelData> StoredListData = InternalSettings.ViewFilters.GetValue();
```

Comment:
This field needs its private access modifier specified explicitly :)

Reply:
Added, should be good to go now